spec/features/admin/enterprises_spec.rb

```diff
@@ -342,6 +342,8 @@
         fill_in 'enterprise_address_attributes_address1', with: 'z'
         fill_in 'enterprise_address_attributes_city', with: 'z'
         fill_in 'enterprise_address_attributes_zipcode', with: 'z'
+        select2_select 'Australia', from: 'enterprise_address_attributes_country_id'
```

Comment:
Does that mean people always have to select Australia even when it's the only country in the database? It would be nice to use `Spree::Config[:default_country_id]` as a default like before.

Reply:
good point. thanks. done.